amqp_job.go

```diff
@@ -21,15 +21,16 @@ type amqpJob struct {
 	payload         *JobPayload
 	rawPayload      *simplejson.Json
 	startAttributes *backend.StartAttributes
+	instanceID      string
 	received        time.Time
 	started         time.Time
 	finished        time.Time
 	stateCount      uint
 }
 
 func (j *amqpJob) GoString() string {
-	return fmt.Sprintf("&amqpJob{conn: %#v, delivery: %#v, payload: %#v, startAttributes: %#v}",
-		j.conn, j.delivery, j.payload, j.startAttributes)
+	return fmt.Sprintf("&amqpJob{conn: %#v, delivery: %#v, payload: %#v, startAttributes: %#v, instance_id: %#v}",
+		j.conn, j.delivery, j.payload, j.startAttributes, j.instanceID)
```

Comment:
this can also be removed, as we are getting the instance id from ctx in `createStateUpdateBody`, no need to attach it to the job